src/ontology/cl-edit.owl

```diff
@@ -28462,6 +28463,15 @@ AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4023171 <http:
 AnnotationAssertion(rdfs:label obo:CL_4023171 "trigeminal motor neuron")
 EquivalentClasses(obo:CL_4023171 ObjectIntersectionOf(obo:CL_0000100 obo:CL_4023169))
 
+# Class: obo:CL_4023181 (hypendymal cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref <https://orcid.org/0000-0001-7258-9596>) Annotation(oboInOwl:hasDbXref "PMID:26778959"^^xsd:string) Annotation(oboInOwl:hasDbXref "PMID:9579598"^^xsd:string) obo:IAO_0000115 obo:CL_4023181 "A neurecto-epithelial cell that is part of the basal layer of the subcommissural organ and specialises in the secretion of proteins into the subarachnoid space. Hypendymal cells have similar characteristics to ependymal cells and expresses SCO-spondin.")
+AnnotationAssertion(<http://purl.org/dc/terms/contributor> obo:CL_4023181 <http://orcid.org/0000-0002-2105-0715>)
+AnnotationAssertion(rdfs:label obo:CL_4023181 "hypendymal cell")
+SubClassOf(obo:CL_4023181 obo:CL_0000710)
+SubClassOf(obo:CL_4023181 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0002139))
```

Comment:
Consider updating text definition of 'subcommissural organ' since it only invokes 'ependymal cells'.

Reply:
Will do - https://github.com/obophenotype/uberon/issues/2605